demisto_sdk/commands/common/tests/pack_metadata_validator_test.py

```diff
@@ -136,3 +137,17 @@ def test_metadata_not_dict(self, mocker):
         mocker.patch.object(validator, '_add_error')
         with pytest.raises(BlockingValidationFailureException):
             assert not validator._is_pack_meta_file_structure_valid()
+
+    @pytest.mark.parametrize('metadata', [
+        os.path.join(FILES_PATH, 'pack_metadata_empty_categories.json')
```

Comment:
why do you use `parametrize` for a single case?

Reply:
Matter of convenience... `FILES_PATH` is a class variable so the alternative is define a local variable inside the function, using self.__class__.FILES_PATH. I replaced it anyway hope It's fine